packages/frontend/app/routes/reports.js

```diff
@@ -1,10 +1,3 @@
 import Route from '@ember/routing/route';
-import { service } from '@ember/service';
 
-export default class ReportsRoute extends Route {
-  @service session;
-
-  beforeModel(transition) {
-    this.session.requireAuthentication(transition, 'login');
```

Comment:
double-checking - is the removal of the auth check here intentional?

Reply:
Nope, must have gotten re-generated and I missed this in my review. Great catch!